allennlp/semparse/worlds/atis_world.py

```diff
@@ -1,19 +1,47 @@
 from copy import deepcopy
-from typing import List, Dict, Tuple
+from typing import List, Dict, Tuple, Set, Callable
+import sqlite3
 import numpy
+from nltk import ngrams
 
 from parsimonious.grammar import Grammar
 
 from allennlp.semparse.contexts.atis_tables import * # pylint: disable=wildcard-import,unused-wildcard-import
 from allennlp.semparse.contexts.sql_table_context import \
-        SqlTableContext, SqlVisitor, generate_one_of_string, format_action
+        SqlTableContext, SqlVisitor, format_action
 
 from allennlp.data.tokenizers import Token, Tokenizer, WordTokenizer
 
+def get_strings_from_utterance(tokenized_utterance: List[Token]) -> Dict[str, List[int]]:
+    """
+    Based on the current utterance, return a dictionary where the keys are the strings in the utterance
```

Comment:
If you read this out of context, you wonder what "the strings" mean.  You mean this to be "strings from the database", but that's not clear anywhere, so it's a bit confusing.